src/build.config.js

```diff
@@ -58,9 +58,18 @@ export default {
         agent:
           process.env.VUE_APP_AGENT_TESTNET_URL || 'https://liquality.io/swap-testnet-dev/agent'
       },
-      liqualityBoost: {
-        name: 'Liquality Boost',
-        type: SwapProviderType.LIQUALITYBOOST,
+      // Native to ERC20
+      liqualityBoost_type1: {
```

Comment:
name is in mixed case "liqualityBoost_type1".
Should be in  camel case for consistency. 
Also I don't really like the name liqualityBoost_type1  and type2 - we should brainstorm/ ask UX team.

Last but not least keep in mind that changing the name of existing liquidity providers will need backup compatibility support.

Reply:
Fixed. Liquality boost in included in `mapLegacyProvidersToSupported`.